docs/publishers-secrets.example.sh

```diff
@@ -29,4 +29,13 @@ export UPHOLD_AUTHORIZATION_ENDPOINT="https://sandbox.uphold.com/authorize/<UPHO
 export UPHOLD_API_URI="https://api-sandbox.uphold.com" # the API endpoint for Uphold.
 export UPHOLD_SCOPE="cards:read,cards:write,user:read"
 export UPHOLD_DASHBOARD_URL="https://sandbox.uphold.com/dashboard"
-export TERMS_OF_SERVICE_URL="https://basicattentiontoken.org/publisher-terms-of-service/"
\ No newline at end of file
+export TERMS_OF_SERVICE_URL="https://basicattentiontoken.org/publisher-terms-of-service/"
+
+# Get these from the Google Web application setup. During setup:
+# - make sure the authorized redirect url goes to the endpoint /publishers/auth/google_oauth2/callback
+export GOOGLE_CLIENT_ID=""
+export GOOGLE_PROJECT_ID=""
+export GOOGLE_AUTH_URI=""
```

Comment:
where do i find GOOGLE_AUTH_URI, GOOGLE_TOKEN_URI and GOOGLE_AUTH_PROVIDER_X509_CERT_URL